.NET/Microsoft.Recognizers.Text.DateTime/Hindi/Extractors/HindiDurationExtractorConfiguration.cs

```diff
@@ -70,7 +70,7 @@ public HindiDurationExtractorConfiguration(IDateTimeOptionsConfiguration config)
 
         public IImmutableDictionary<string, long> UnitValueMap { get; }
 
-        bool IDurationExtractorConfiguration.CheckBothBeforeAfter => DateTimeDefinitions.CheckBothBeforeAfter;
+        bool IDurationExtractorConfiguration.CheckBothBeforeAfter => true;
```

Comment:
This should be read from the YAML file.

Reply:
If this flag only applies to Durations, not in general, please declare different variables in the YAML.

Reply:
ok, modified to use the global flag in the yaml. If necessary, we will introduce subtype specific flags in future PRs (in previous tests, the use of the global flag broke other subtypes, but at the moment everything works)